drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -237,15 +237,35 @@
     background-color: #fff;
 }
 
+@media screen and (max-width: 480px) {
+    .xblock--drag-and-drop .target {
+        overflow: auto;
+        display: block;
+    }
+
+    .xblock--drag-and-drop .target-img {
+        max-width: none;
+    }
+}
+
+
+@media screen and (min-width: 481px) {
+    /* Style for non-mobile */
```

Comment:
@cgopalan Style for non mobile should just go under default styles (no need to wrap into `(min-width: 481px)` media query). You already reset `max-width` to `none` in mobile-only styles. You can also unset the default `width: 100%` using `width: auto` in mobile styles.

Reply:
@mtyaka I tried this, but if I use `width : auto` and remove the non-mobile style, the target image behaves like before. This is because for some reason the mobile css does not override the non-mobile css. See screenshot. Any idea why?

<img width="682" alt="screen shot 2017-09-28 at 11 43 17 am" src="https://user-images.githubusercontent.com/395294/30976491-ead9fa84-a442-11e7-8936-d21052e9cf56.png">

Reply:
@cgopalan I think it's because the mobile styles are defined *above* the default styles. If you move the mobile styles below, it should override correctly.

Reply:
@mtyaka please ignore the above comment. It turned out I was defining the mobile style before the default style, so the mobile style always got overriden. Its working now.

Reply:
@mtyaka I think we both typed the responses at the same time! Thanks for confirming.